pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -38,20 +38,56 @@ def test_delta_to_tick():
 # ---------------------------------------------------------------------
 
 
-def test_Hour():
-    assert_offset_equal(Hour(),
-                        datetime(2010, 1, 1), datetime(2010, 1, 1, 1))
-    assert_offset_equal(Hour(-1),
-                        datetime(2010, 1, 1, 1), datetime(2010, 1, 1))
-    assert_offset_equal(2 * Hour(),
-                        datetime(2010, 1, 1), datetime(2010, 1, 1, 2))
-    assert_offset_equal(-1 * Hour(),
-                        datetime(2010, 1, 1, 1), datetime(2010, 1, 1))
-
-    assert Hour(3) + Hour(2) == Hour(5)
-    assert Hour(3) - Hour(2) == Hour()
-
-    assert Hour(4) != Hour(1)
+class TestHour(object):
+    def test_construct_hour3(self):
+        # smoketest to debug appveyor segfault
+        Hour(3)
+
+    def test_construct_hour2(self):
+        # smoketest to debug appveyor segfault
+        Hour(2)
+
+    def test_construct_hour5(self):
+        # smoketest to debug appveyor segfault
+        Hour(5)
+
+    def test_hour3_add_hour2(self):
+        # smoketest to debug appveyor segfault
+        Hour(3) + Hour(2)
+
+    # def test_hour_add_hour(self):
+    #    assert Hour(3) + Hour(2) == Hour(5)
+
+    def test_hour_sub_hour(self):
+        assert Hour(3) - Hour(2) == Hour()
+
+    def test_hour_neq_hours(self):
+        assert Hour(4) != Hour(1)
+
+    def test_neg_hour(self):
+        assert_offset_equal(-1 * Hour(),
+                            datetime(2010, 1, 1, 1), datetime(2010, 1, 1))
+
+    def test_negative_n(self):
+        assert_offset_equal(Hour(-1),
+                            datetime(2010, 1, 1, 1),
+                            datetime(2010, 1, 1))
+
+    def test_one_hour(self):
```

Comment:
I wonder if you need to move ``__eq__`` to ``_BaseOffset``

Reply:
For lack of any better ideas, let's give it a shot.

Reply:
If somehow this does fix the error, pls don't merge it just yet.  I'm trying to avoid moving methods to the cython classes until I've vetted them, and `__eq__` is very much _not_ OK.